src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json

```diff
@@ -0,0 +1,37 @@
+{
+  "symbol": "iOWN",
+  "name": "iOWN Token",
+  "type": "ERC20",
+  "address": "0x555d051538c7a13712f1f590fa6b4c176ca4529f",
+  "ens_address": "",https://www.youtube.com/channel/UC3gsYQsL0yYC_ShOYUGP5WA
```

Comment:
Please remove the youtube link

Reply:
@gamalielhere hi, can you please check if we can get listed now?

Reply:
yup , will merge and add

Reply:
Thanks @gesseekur